src/main/java/io/singularitynet/Client/VereyaModClient.java

```diff
@@ -243,9 +245,13 @@ private void onKey(long window, int key, int scancode, int action, int modifiers
         // do default thing if any screen is open
         Screen screen = MinecraftClient.getInstance().currentScreen;
         if (screen != null) {
-            if (screen instanceof ChatScreen) {
-                return;
-            }
+            wasNotNull = true;
+            return;
+        }
+        if (wasNotNull)
+        {
+            setInputType(inputType);
```

Comment:
why you set the same input type here?

Reply:
@noskill  because we want inputType to be **exactly** the same. As I've already said, this PR is a stub-fix for this issue https://github.com/trueagi-io/minecraft-demo/issues/164

Reply:
I don't get it, you are just setting the current input type here, why to set it again if it's already set?

Reply:
by the way there is relevant code in VereyaModClient.interact